pallets/gear/src/tests.rs

```diff
@@ -827,19 +844,17 @@ fn delayed_send_program_message_with_reservation() {
         run_to_next_block(None);
 
         assert_eq!(Balances::reserved_balance(&USER_1), 0);
-        assert_eq!(
-            free_balance - delay_holding_fee + reserve_for_fee,
-            Balances::free_balance(&USER_1)
-        );
     }
 
     init_logger();
 
-    for i in 1..4 {
+    for i in 1..4{
         new_test_ext().execute_with(|| scenario(i));
     }
 }
 
+// TODO mikita test where reserved less than dispathc hold fee
```

Comment:
todo

Reply:
in test delayed_send_program_message_with_low_reservation